ui/ducks/send/send.js

```diff
@@ -2046,7 +2047,14 @@ export function updateSendAmount(amount) {
     await dispatch(
       addHistoryEntry(`sendFlow - user set amount to ${logAmount}`),
     );
-    await dispatch(actions.updateSendAmount(amount));
+    if (
+      draftTransaction.asset.type === AssetType.NFT &&
+      draftTransaction.asset.details.standard === TokenStandard.ERC1155
+    ) {
+      await dispatch(actions.updateSendAmount(toHex(amount)));
+    } else {
+      await dispatch(actions.updateSendAmount(amount));
```

Comment:
the amount value received by the user is in decimal in the case of an `erc1155`, but the draftTransaction state must receive a hexadecimal value.
so, when the user enters 18 (decimal value), it will be read as `24`, as the hexadecimal value of `24` is `0x18`.